.github/workflows/argilla.docs.yml

```
@@ -55,16 +57,17 @@ jobs:
         run: echo "${{ github.ref }}"
           echo "${{ github.head_ref }}"
 
+      - run: |
+          pdm run mike deploy latest --push
+          pdm run mike set-default latest
```

```suggestion
          pdm run mike deploy develop --push
```
I would maybe just call the latest version of the docs/develop. I think we should try avoid the flow you had because merging new feature into main might then cause in doc updates without the supported feature.

---

We still have 2 branches: `develop` and `main`. The one for continuous development is `develop` and the one with the stable version of argilla is `main`.  So `develop` should generate a `dev` version of the docs, and `main` update the latest release. We can also generate tagged versions when a new tag is created (should be created from the main branch). And we can define more rules to protect merges into the main branch.

---

This step applies only to the main branch (not develop)